src/pages/admin/CreateNewMeetingPage.vue

```diff
@@ -0,0 +1,115 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import apis, { CreateMeeting, Meeting } from '@/lib/apis'
+import BaseInput from '@/components/UI/BaseInput.vue'
+import BaseButton from '@/components/UI/BaseButton.vue'
+import EmoineHeader from '@/components/EmoineHeader.vue'
+import router from '@/router'
+
+//todo: 専用ヘッダー画像のアップロード
+const headerLogo = '/logo.png'
+
+const liveUrl = ref('')
+const isUrlValidated = ref(true)
+
+function getLiveIdFromUrl(): string {
+  const params = new URL(liveUrl.value).searchParams
+  if (params.has('v')) return
+  return params.get('v')
+}
+
+const createMeeting = async () => {
+  let id = ''
+  try {
+    const meetingInfo: CreateMeeting = {
+      video_id: getLiveIdFromUrl(),
+      description: ''
+    }
+    const res = (await apis.createMeeting(meetingInfo)).data
+    id = res.id
+  } catch (e: unknown) {
+    console.log(e)
+    isUrlValidated.value = false
+    return
+  }
```

Comment:
こうすると`throw new Error('Invalid URL')`ってしたときに`e.message`で`'Invalid URL'`を取り出せたはず

```suggestion
  } catch (e: unknown) {
    if (e instanceof Error) {
      console.log(e.message)
      isUrlValidated.value = false
      return
    }
  }
```